phpstan.neon.dist

```diff
@@ -0,0 +1,19 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    level: 0
+    bootstrapFiles:
+        - vendor/bin/.phpunit/phpunit/vendor/autoload.php
+    paths:
+        - src
+        - tests
+    excludePaths:
+        - src/Controller/Api/*
```

Comment:
It would be better to avoid adding those 3 paths to the exclude path.

Reply:
Solved